application/templates/pdb.yaml

```diff
@@ -1,15 +1,17 @@
-{{- if and (.Values.pdb).enabled (or .Values.pdb.minAvailable .Values.pdb.maxUnavailable) }}
-{{ if $.Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" -}}
+{{- if or .Values.pdb.minAvailable .Values.pdb.maxUnavailable | and .Values.pdb.enabled }}
+
+{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
 apiVersion: policy/v1
-{{- else -}}
+{{- else }}
 apiVersion: policy/v1beta1
 {{- end }}
+
```

Comment:
I added the empty lines precisely because of readability. Many disagree with me on this, but YAML starts looking very messy to me when no empty lines separate sections of it.

Reply:
I agree :)